packages/drivers/odsp-driver/src/vroom.ts

```diff
@@ -38,8 +38,9 @@ interface IJoinSessionBody {
  * @param options - Options to fetch the token.
  * @param disableJoinSessionRefresh - Whether the caller wants to disable refreshing join session periodically.
  * @param isRefreshingJoinSession - whether call is to refresh the session before expiry.
- * @param guestDisplayName - display name used to identify guest user joining a session.
- * This is optional and used only when collab session is being joined via invite.
+ * @param displayName - display name used to identify client joining a session.
+ * This is optional and used only when collab session is being joined in app-only mode.
```

Comment:
What is app-only mode?

Reply:
Also, can we add this same detailed explanation to the place where the argument is defined in HostStoragePolicy?

Reply:
Client is in app only mode when there is no user context.
I will add more comments.